### PR TITLE
replaces repl iframe with new sandbox

### PR DESCRIPTION
**Pull request by graciemcguire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,16 +66,19 @@ ReferenceError: maximumSpeed is not defined
...
+`maximumSpeed = 9000`
 
-Notice that `maximumSpeed = 9000`, the assignment expression, evaluates to
-`9000` when run. Once `maximumSpeed` is defined, JavaScript will know what
-it is. (We'll look at this more closely in the next lesson.)
+Notice that by using the assignment expression, `maximumSpeed = 9000`, our code
+evaluates to `9000` when run. Once `maximumSpeed` is defined, JavaScript will
+know what it is. Now try putting just `maximumSpeed` in and hitting enter and
+you'll see that JavaScript remembered it's value! (We'll look at this more
+closely in the next lesson.)
```
Comment (lizbur10) [inline README.md:84]: should be "remembered its value"

File: README.md [hunk 2]
```diff
@@ -91,7 +94,7 @@ to" can be changed during the running of the program. Being able to change the
 value a variable points to is very important. For example, if we need to do
 something 10 times, we need a variable to keep track of how many times the thing
 happens. That variable will need to change: its value will need to increase by 1
-each time. Here's "mutability" in action:
+each time. Here's _"mutability"_ in action:
```
Comment (lizbur10) [inline README.md:103]: I don't think we need both italics and quotes - let's go with the italics

File: README.md [hunk 3]
```diff
@@ -107,7 +110,7 @@ heightInCentimeters = 180;
 
 We can try these out in REPL:
```
Comment (lizbur10) [inline README.md:122]: I think we need to walk them through this: enter the first line, see that the return value is 50, enter the second, see that the return value is 180, then just run `heightInCentimeters` to see that the latest value is saved

File: README.md [hunk 4]
```diff
@@ -66,16 +66,19 @@ ReferenceError: maximumSpeed is not defined
 ...followed by many lines starting with `at...`. Here, JavaScript, by default,
 doesn't know anything about `maximumSpeed`.
 
-<iframe height="400px" width="100%" src="https://repl.it/@MaxwellBenton2/OrderlyDeadParticle?lite=true" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
+<iframe height="400px" width="100%" src="https://replit.com/@lizbur10/Sandbox?lite=1&outputonly=1" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
```
Comment (lizbur10) [inline README.md]: This repl window isn't really doing anything here, now that it doesn't contain code. I suggest either changing the part just before this to have students enter `maximumSpeed;` to see what happens, or move the repl after the instructions on lines 72-73

**Review by lizbur10 — COMMENTED** [review]
I added a couple of suggestions for making things a little more explicit, now that they don't have the existing code in the repl to orient them

File: README.md (hunk 3 above)
Comment (graciemcguire) [reply README.md:122]: How does this look?

Comment (lizbur10) [reply README.md:122]: Perfect!

File: README.md [hunk 5]
```diff
@@ -91,23 +97,31 @@ to" can be changed during the running of the program. Being able to change the
...
-each time. Here's "mutability" in action:
+each time. Here's _mutability_ in action:
+
+Let's test try out in REPL:
+
+<iframe height="400px" width="100%" src="https://replit.com/@lizbur10/Sandbox?lite=1&outputonly=1" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
 
-Many years ago my height in centimeters was:
+Many years ago my height in centimeters was 50cm, go ahead and add the following
+line into the console:
```
Comment (lizbur10) [inline README.md:107]: Use a semicolon here rather than a comma

**Review by lizbur10 — CHANGES_REQUESTED** [review]
Looks great! Just one minor correction.

**Pull request merged**